elasticsearch/values.yaml

```diff
@@ -92,6 +92,10 @@ extraInitContainers: []
   #   image: busybox
   #   command: ['do', 'something']
 
+# This is the PriorityClass settings as defined in
+# https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+priorityClassName: {}
```

Comment:
I think this should be an empty string looking at https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority.